src/main/webapp/utilities.js

```diff
@@ -90,3 +90,23 @@ function getDateInLocalTimeZone(dateObject = new Date()) {
   const newTime = dateObject.getTime() - dateObject.getTimezoneOffset()*60*1000;
   return new Date(newTime);
 }
+
+/**
+ * Given a list of words, encode them for a query string with commas separating
+ * the values.
+ *
+ * @param {string[]} listOfValues list of words that should be encoded
+ * @param {boolean} surroundWithQuotes true if words should be surrounded with
+ *     quotes, false if the words can be encoded as is
+ * @return {string} the value for a query parameter, with each value seperated
+ *     by a comma.
+ */
+function encodeListForUrl(listOfValues, surroundWithQuotes) {
```

Comment:
Seems like this function is called once with `surroundWithQuotes = true`, would there be a case where this is used with `surroundWithQuotes = false`?

Reply:
Good catch - probably not. If for some reason in the future this does arise, it can be developed then. I'm removing this :)

Reply:
This isn't really for me, but for others - if anyone else wants to make a list but the words don't need to be surrounded by quotes (i.e. `<some url>?list=phrase one,phrase two` is fine instead of `<some url>?list="phrase one","phrase two"`) they can still use this utility that way

Reply:
Hmm okay, unless there is some use case where it is a must to have no quotes around the words, it might be better to just stick with the one with quotes throughout the entire codebase. Though this thought is only because I can't think of a use case, let me know if there is!

Reply:
+1 to always using quotes

Reply:
No worries, changing over. 

If this is needed in the future, it's easy enough to implement :)